docs/administrator-documentation/moderne-dx/how-to-guides/dx-org-service.md

```diff
@@ -0,0 +1,77 @@
+---
+sidebar_label: Creating an organizations service
+description: How to create and configure an organizations service that pairs with Moderne DX.
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Creating an organizations service
+
+You should create a dedicated organizations service if you want to:
+
+* Limit access to the organizations you've [previously defined](./configure-dx-organizations.md) so that some users only have access to some repositories OR
+* Customize commit messages by repository (e.g., adding a JIRA ticket to your commit messages based on the repository)
+
+This guide will walk you through everything you need to know to create such a service.
+
+#### Prerequisites
+
+This guide assumes that:
```

Comment:
nit (meaning feel free to directly resolve the conversation if you disagree): I was always taught to only use a colon before a list if it's after an independent clause.

```suggestion
This guide assumes the following:
```